tox.ini

```diff
@@ -40,7 +40,7 @@ setenv =
     lsof: _PYTEST_TOX_POSARGS_LSOF=--lsof
 
     pexpect: _PYTEST_TOX_PLATFORM=linux|darwin
-    pexpect: _PYTEST_TOX_POSARGS_PEXPECT=testing/test_pdb.py testing/test_terminal.py testing/test_unittest.py
+    pexpect: _PYTEST_TOX_POSARGS_PEXPECT=-m uses_pexpect
```

Comment:
Switched to using the new marker here.

Reply:
Meh.. that will fail when combined with twisted etc I guess.
Also with using pexpect for macos in the other PR.
Should get back to https://github.com/pytest-dev/pytest/pull/5313 first...

Reply:
...it is not being combined though, so that should be good.

Reply:
Pulled that into the fix-pexpect PR (6896dbc).